scripts/install.sh

```diff
@@ -40,6 +40,15 @@ case "$KERN" in
     *) ;;
 esac
 
+OLLAMA_VERSION=${OLLAMA_VERSION:-""}
+if [ -z "${OLLAMA_VERSION}" ]; then
+    URL_PREFIX="https://ollama.com/download"
+else
+    # URL_PREFIX="https://github.com/ollama/ollama/releases/download/v${OLLAMA_VERSION}"
+    # DO NOT MERGE - for testing only...
+    URL_PREFIX="https://github.com/dhiltgen/ollama/releases/download/v${OLLAMA_VERSION}"
```

Comment:
Remove this before we merge - using this to test now to ensure the dependency tar file is present.